patt/src/main/java/org/example/strategy/CarRouteStrategy.java

```diff
@@ -0,0 +1,8 @@
+package org.example.strategy;
+
+public class CarRouteStrategy implements RouteStrategy{
+    @Override
+    public String calculateRoute(String destination) {
+        return destination= "CarRoute";
```

Comment:
N: just return some value.
It's a bad practice to modify arguments.
https://chat.openai.com/share/e2185127-e066-4d5f-8d50-bdb8c1a87e51